CITATION.cff

```diff
@@ -0,0 +1,25 @@
+cff-version: 1.2.0
+message: "If you use Volcano in your work, please cite it using the information below. This helps us demonstrate the project's impact."
+title: "Volcano: A Cloud Native Batch System"
+authors:
+  - given-names: "Klaus"
+    family-names: "Ma"
+  - given-names: "Kevin"
+    family-names: "Wang"
+  - name: "The Volcano Community & Contributors"
+    type: entity
+
+version: "1.11.2"
```

Comment:
What if we release new versions in the future, should we also change this after new releasing?

Reply:
yes, you are right, its a thing to consider, i would say, let me remove version and set the date to the initial project date (or even remove it). then in the notes, I'll add the link to releases (https://github.com/volcano-sh/volcano/releases). what do you think?

Reply:
I added notes with reference to release page.

Reply:
Ok.